torch_patches/.torch_pin

```diff
@@ -0,0 +1 @@
+#117182
```

Comment:
please follow the example of https://github.com/pytorch/xla/blob/r2.1/.torch_pin. I think we should just pin it to releases/2.2 branch for now and update it to a tag after release it out.

Reply:
do we wanna update to `tags/v2.2.0-rc8` directly?

Reply:
Thank you, changed!